actions/views/rhs.js

```diff
@@ -280,6 +280,34 @@ export function selectPost(post) {
     };
 }
 
+export function updatePreviewOnCreateComment(showPreview) {
+    return {
+        type: ActionTypes.UPDATE_SHOW_PREVIEW_ON_CREATE_COMMENT,
```

Comment:
Similarly, `SET_SHOW_PREVIEW_ON....`. However, another option might just be to make this all explicit and have a `SHOW_PREVIEW_ON_CREATE_COMMENT` and `HIDE_PREVIEW_ON_CREATE_COMMENT`.